### ISSUE TITLE
Conflicting with other mods that have flight

### ISSUE THREAD

**Issue by LunaFolf** [ArtanMod/DeepMobLearningReloaded#1]
I'm currently playing on `1.20.1` with my modpack, and I'm experiencing an issue.
When using Mystical Agriculture's Awakened supremium armor, with a flight augment module, flight is allowed however switching items, dopping items, taking damage etc causes the flight to be disabled and the player falls out of the sky.

I believe the issue stems from this even handler: https://github.com/ArtanMod/DeepMobLearningReloaded/blob/e15e402331daf1e8519227237dfc82974650a3ca/src/main/java/jp/artan/dmlreloaded/common/events/PlayerHandler.java#L31C81-L31C81

The function seems to be checking if the player is wearing a fullset of glitch armor, and if not disables their flight.
The issue is this breaks other mods using the same method of flying.

My recommendation would be to keep a record of the previous event's armor, and only disable flight if the player was wearing glitch armor to begin with.
This is just a 5am guess though, there may be better ways of fixing this.

**Comment by Xplodin** [comment]
Yeah this bug existed in deep mob learning reforged and is here too
